src/CMakeLists.txt

```diff
@@ -5,25 +5,27 @@ set(MAIN_SRC_FILES CovidSim.cpp BinIO.cpp Rand.cpp Error.cpp Dist.cpp Kernels.cp
 set(MAIN_HDR_FILES CovidSim.h BinIO.h Rand.h Constants.h Country.h Error.h Dist.h Kernels.h Bitmap.h Model.h Param.h SetupModel.h ModelMacros.h InfStat.h CalcInfSusc.h Sweep.h Update.h MicroCellPosition.hpp Direction.hpp InverseCdf.h)
 source_group(covidsim\\main FILES ${MAIN_SRC_FILES} ${MAIN_HDR_FILES})
 
+add_subdirectory(Geometry)
+
 # CovidSim target
 add_executable(CovidSim ${MAIN_SRC_FILES} ${MAIN_HDR_FILES})
 target_include_directories(CovidSim PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
 
-add_subdirectory(Geometry)
 add_subdirectory(Models)
 
 if(USE_OPENMP)
-  target_link_libraries(CovidSim PUBLIC OpenMP::OpenMP_CXX)
+  target_link_libraries(CovidSim PUBLIC geometrylib OpenMP::OpenMP_CXX)
```

Comment:
I think multiple `target_link_libraries` calls are additive - so we only need one here.

Reply:
You would appear to be correct as the tests pass. Thanks.